filmfest/static/logo.svg

```diff
@@ -31,7 +32,7 @@
      showgrid="true"
      inkscape:document-units="px"
      inkscape:grid-bbox="true"
-     inkscape:window-width="1364"
+     inkscape:window-width="1280"
      inkscape:window-height="747"
```

Comment:
It is height and width of my Desktop. Inkspace records workspace parameters in the file.